openssh.yaml

```diff
@@ -100,6 +102,17 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/usr/lib/ssh
           mv "${{targets.destdir}}"/usr/lib/ssh/sftp-server "${{targets.subpkgdir}}"/usr/lib/ssh/
 
+  - name: "openssh-pam"
```

Comment:
The only things that we need to add are the right config files after compiling openssh with pam support. There's two involved: The `sshd.pam` pam config which needs to be created for the deployed OS and the other is an sshd flag flip to tell sshd to use pam as an auth mechanism. There aren't however any other binaries we need to install.